kpatch/kpatch

```diff
@@ -165,7 +165,27 @@ load_module () {
 	fi
 
 	echo "loading patch module: $module"
-	insmod "$module"
+	local i=0
+	while true; do
+		out=$(insmod "$module" 2>&1)
+		[[ -z $out ]] && break
+		echo $out 1>&2
+		[[ ! $out =~ "Device or resource busy" ]] &&
+			die "failed to load module $module"
+
+		# "Device or resource busy" means the activeness safety check
+		# failed.  Retry in a few seconds.
+		i=$((i+1))
+		if [[ $i = 5 ]]; then
```

Comment:
`=` is legit because all variables are really strings in bash.  So it's testing for the string "5".  `-eq` would have worked too.